org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -1362,6 +1361,9 @@ public void diagnosticsWithWebApp() throws BadLocationException {
 				"</web-app>";
 		Diagnostic diagnostic = d(4, 5, 4, 12, XMLSchemaErrorCode.cvc_complex_type_2_4_b);
 		testDiagnosticsFor(xml, "src/test/resources/catalogs/catalog-web-app.xml", diagnostic);
+		testCodeActionsFor(xml, diagnostic, "src/test/resources/catalogs/catalog-web-app.xml",

```

Comment:
I'm not sure why the `testCodeActionsFor` test returns no code action when it works in the demo. (Expected [1] but was [0])